doc/source/whatsnew/v2.1.0.rst

```diff
@@ -496,6 +496,7 @@ Plotting
 
 Groupby/resample/rolling
 ^^^^^^^^^^^^^^^^^^^^^^^^
+- Bug in :meth:`DataFrame.resample` and :meth:`Series.resample` :class:`Datetimelike` ``origin`` has no effect in resample for ``MS`` frequency (:issue:`53662`)
```

Comment:
is it just `MS` frequency that was the issue?

Reply:
I believe this impacts all frequencies.